data/cpupower-preferences.glade

```diff
@@ -548,7 +551,76 @@ Jimmy Scionti (@amivaleo)</property>
       </object>
       <packing>
         <property name="left_attach">1</property>
+        <property name="top_attach">5</property>
+      </packing>
+    </child>
+    <child>
+      <object class="GtkGrid" id="ProfileSettingsAutoSwitchGrid">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="margin_right">80</property>
+        <property name="margin_top">12</property>
+        <property name="row_spacing">8</property>
+        <property name="column_spacing">8</property>
+        <child>
+          <object class="GtkLabel" id="ProfileDefaultACLabel">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="halign">end</property>
+            <property name="label" translatable="yes">Default AC profile</property>
+          </object>
+          <packing>
+            <property name="left_attach">0</property>
+            <property name="top_attach">0</property>
+          </packing>
+        </child>
+        <child>
+          <object class="GtkLabel" id="ProfileDefaultBatLabel">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="halign">end</property>
+            <property name="label" translatable="yes">Default battery profile</property>
+          </object>
+          <packing>
+            <property name="left_attach">0</property>
+            <property name="top_attach">1</property>
+          </packing>
+        </child>
+        <child>
+          <object class="GtkCheckButton" id="ProfileDefaultBatCheck">
+            <property name="visible">True</property>
+            <property name="can_focus">True</property>
+            <property name="receives_default">False</property>
+            <property name="tooltip_text" translatable="yes">Set as default profile for battery operation</property>
+            <property name="use_stock">True</property>
+            <property name="draw_indicator">True</property>
+            <signal name="toggled" handler="onProfileDefaultBatChecktoggled" swapped="no"/>
+          </object>
+          <packing>
+            <property name="left_attach">1</property>
+            <property name="top_attach">1</property>
+          </packing>
+        </child>
+        <child>
+          <object class="GtkCheckButton" id="ProfileDefaultACCheck">
+            <property name="visible">True</property>
+            <property name="can_focus">True</property>
+            <property name="receives_default">False</property>
+            <property name="tooltip_text" translatable="yes">Set as standard profile for plugged in operation</property>
```

Comment:
Very good! We should have tooltips on all our controls to make the UI more accessible :see_no_evil: